google-cloud-spanner/src/test/java/com/google/cloud/spanner/BaseSessionPoolTest.java

```diff
@@ -54,7 +54,10 @@ public ScheduledExecutorService get() {
 
     @Override
     public void release(ScheduledExecutorService executor) {
-      executor.shutdown();
+      try {
```

Comment:
This cleans up errors that are being logged during tests, clogging up the test log.